### ISSUE TITLE
Cannot hit continue to begin installer on Mac OS 10.8.5

### ISSUE THREAD

**Issue by shoshi** [docker/toolbox#134]
Steps to reproduce:

1. Download the latest available version from: https://www.docker.com/toolbox. `DockerToolbox-1.8.1c.pkg`
2. Launch the installer.
3. Click "Continue".
![screen shot 2015-08-31 at 1 35 19 pm](https://cloud.githubusercontent.com/assets/171493/9585494/55ce47dc-4fe5-11e5-8ecc-b00f5e06dd21.png)
4. Get to the Introduction step and click "Continue" again.
![screen shot 2015-08-31 at 1 35 27 pm](https://cloud.githubusercontent.com/assets/171493/9585502/5bf2d786-4fe5-11e5-85d9-11bb3b0b2a6c.png)
5. Get stuck here. No options are available.
![screen shot 2015-08-31 at 1 35 32 pm](https://cloud.githubusercontent.com/assets/171493/9585510/6d0cee26-4fe5-11e5-8453-a84e136c873c.png)

The only errors in the installer log seem to be related to Mixpanel:
```
Aug 31 13:09:48 *.local Installer[5386]: objc[5386]: Class Mixpanel is implemented in both /var/folders/8t/218drryn6b3_xnrgzy0pnxlc0000gn/T/com.apple.install.3bOAli6T/overviewplugin.bundle/Contents/MacOS/overviewplugin and /var/folders/8t/218drryn6b3_xnrgzy0pnxlc0000gn/T/com.apple.install.3bOAli6T/managevmplugin.bundle/Contents/MacOS/managevmplugin. One of the two will be used. Which one is undefined.
Aug 31 13:09:48 *.local Installer[5386]: objc[5386]: Class Mixpanel is implemented in both /var/folders/8t/218drryn6b3_xnrgzy0pnxlc0000gn/T/com.apple.install.3bOAli6T/overviewplugin.bundle/Contents/MacOS/overviewplugin and /var/folders/8t/218drryn6b3_xnrgzy0pnxlc0000gn/T/com.apple.install.3bOAli6T/quickstartplugin.bundle/Contents/MacOS/quickstartplugin. One of the two will be used. Which one is undefined.
```

I also tried rebooting the computer and re-downloading the installer just to see if that helped. Unfortunately, it didn't change anything.

**Comment by jeffdm** [comment]
Thanks for the issue. If you open `Console.app`, are there any entries that seem related there?

**Comment by shoshi** [comment]
Yep, there seem to be! They might even be slightly more helpful.

```
Aug 31 17:41:24 *.local Installer[1660]: CoreAnimation: warning, deleted thread with uncommitted CATransaction; set CA_DEBUG_TRANSACTIONS=1 in environment to log backtraces.
Aug 31 17:41:27 *.local Installer[1660]: -[NSConcreteMutableData base64EncodedStringWithOptions:]: unrecognized selector sent to instance 0x7fe1e3a969e0
Aug 31 17:41:27 --- last message repeated 1 time ---
Aug 31 17:41:27 *.local Installer[1660]: (
		0   CoreFoundation                      0x00007fff92c17b06 __exceptionPreprocess + 198
		1   libobjc.A.dylib                     0x00007fff8deaf3f0 objc_exception_throw + 43
		2   CoreFoundation                      0x00007fff92cae40a -[NSObject(NSObject) doesNotRecognizeSelector:] + 186
		3   CoreFoundation                      0x00007fff92c0602e ___forwarding___ + 414
		4   CoreFoundation                      0x00007fff92c05e18 _CF_forwarding_prep_0 + 232
		5   overviewplugin                      0x000000010cc5df03 +[Mixpanel trackEvent:forPane:withProperties:] + 1149
		6   overviewplugin                      0x000000010cc5e0bd +[Mixpanel trackEvent:forPane:] + 53
		7   overviewplugin                      0x000000010cc5e319 -[overviewpluginPane willEnterPane:] + 402
		8   Installer                           0x0000000102f02a1f Installer + 96799
		9   Installer                           0x0000000102f03b6f Installer + 101231
		10  Installer                           0x0000000102efc547 Installer + 70983
		11  AppKit                              0x00007fff8cd42959 -[NSApplication sendAction:to:from:] + 342
		12  AppKit                              0x00007fff8cd427b7 -[NSControl sendAction:to:] + 85
		13  AppKit                              0x00007fff8cd426eb -[NSCell _sendActionFrom:] + 138
		14  AppKit                              0x00007fff8cd40bd3 -[NSCell trackMouse:inRect:ofView:untilMouseUp:] + 1855
		15  AppKit                              0x00007fff8cd40421 -[NSButtonCell trackMouse:inRect:ofView:untilMouseUp:] + 504
		16  AppKit                              0x00007fff8cd3fb9c -[NSControl mouseDown:] + 820
		17  AppKit                              0x00007fff8cd3750e -[NSWindow sendEvent:] + 6853
		18  AppKit                              0x00007fff8cd33644 -[NSApplication sendEvent:] + 5761
		19  AppKit                              0x00007fff8cc4921a -[NSApplication run] + 636
		20  AppKit                              0x00007fff8cbedbd6 NSApplicationMain + 869
		21  Installer                           0x0000000102eec494 Installer + 5268
		22  libdyld.dylib                       0x00007fff8e8e47e1 start + 0
	)
 ```

**Comment by jeffdm** [comment]
Definitely helpful! Thank you! Taking a look.

**Comment by jeffdm** [comment]
In the meantime, installing via command line should work for you:

`sudo installer -pkg DockerToolbox-1.8.1c.pkg -target /`

**Comment by shoshi** [comment]
Thanks @jeffdm that worked wonderfully!

**Comment by datmark** [comment]
@jeffdm I am getting this when using the command you provided...

```bash
installer: Package name is Docker Toolbox
installer: Upgrading at base path /
installer: The upgrade failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)
```

**Comment by shoshi** [comment]
@mchiang0610 This does seem to be fixed in `DockerToolbox-1.8.2c.pkg`. Thanks!
